src/modules/core/components/OmniPicker/AdditionalOmniPicker.tsx

```diff
@@ -0,0 +1,277 @@
+import { MessageDescriptor, FormattedMessage } from 'react-intl';
+import React, {
+  Component,
+  KeyboardEvent,
+  SyntheticEvent,
+  MouseEvent,
+} from 'react';
+import cx from 'classnames';
+
+import {
+  Choose,
+  OmniPickerData,
+  EmptyRenderFnType,
+  ItemRenderFnType,
+} from './types';
+import OmniPickerContent from './OmniPickerContent';
+import OmniPickerItemEmpty from './OmniPickerItemEmpty';
+import styles from './OmniPicker.css';
+
+interface Appearance {
+  position: 'top' | 'bottom';
+}
+
+export interface Props {
+  /** Appearance object */
+  appearance: Appearance;
+
+  /** Render prop to use for the empty state */
+  renderEmpty: EmptyRenderFnType;
+
+  /** Render prop for rendering the item component */
+  renderItem: ItemRenderFnType<any>;
+
+  /**  Called after the picker was opened */
+  onOpen?: () => void;
+
+  /** Called after the picker was closed */
+  onClose?: () => void;
+
+  /** Called when the picker is being reset */
+  onReset?: () => void;
+
+  /** Passes through the keyUp event from the input field */
+  onKeyUp?: (evt: KeyboardEvent<HTMLElement>) => void;
+
+  /** Passes through the keyDown event from the input field */
+  onKeyDown?: (evt: KeyboardEvent<HTMLElement>) => void;
+
+  /** Passes through the focus event from the input field */
+  onFocus?: (evt: SyntheticEvent<HTMLInputElement>) => void;
+
+  /** Passes through the change event from the input field */
+  onChange?: (evt: SyntheticEvent<HTMLInputElement>) => void;
+
+  /** Passes through the blur event from the input field */
+  onBlur?: (
+    evt:
+      | MouseEvent
+      | SyntheticEvent<HTMLInputElement>
+      | KeyboardEvent<HTMLElement>,
+  ) => void;
+
+  /** Is called when a data entry was picked */
+  onPick?: (itemData: OmniPickerData) => void;
+
+  /** Title for the dropdown header */
+  title?: string | MessageDescriptor;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  choose: Choose;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  close?: (
+    evt:
+      | MouseEvent
+      | SyntheticEvent<HTMLInputElement>
+      | KeyboardEvent<HTMLElement>,
+  ) => void;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  getItem: (data: OmniPickerData[], selectedIdx: number) => OmniPickerData;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  filteredData: OmniPickerData[];
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  id: string;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  inputRef: HTMLElement | null;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  keyUsed: boolean;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  select: (idx: number) => void;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  selected: number;
+
+  /** @ignore Will be injected by `withOmniPicker` */
+  triggerElement: HTMLElement | null;
+}
+
```

Comment:
This component is missing displayName, please, add it. It should be just `OmniPicker.AdditionalOmniPicker`.
I know that the initial component doesn't have it but it's an oversight on our part.